addons/hr_hourly_cost/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     'name': 'Employee Hourly Wage',
     'version': '1.0',
-    'category': 'Services/Employee Hourly Cost',
+    'category': 'Human Resources/Employees',
```

Comment:
Seems opinionated to me

Reply:
hi @Julien00859 ,
to make sure i understand it clearly, let me revert back this to original category or fine to keep as this ?
thanks

Reply:
Actually I don't know, I would like the people from @odoo/rd-services to give their opinion

Reply:
@IT-Ideas updated the pr by reverting the category back to original